addons/linecompass/config.cpp

```diff
@@ -5,7 +5,7 @@ class CfgPatches {
         units[] = {};
         weapons[] = {};
         requiredVersion = REQUIRED_VERSION;
-        requiredAddons[] = {"diwako_dui_main", "diwako_dui_radar"};
+        requiredAddons[] = {"diwako_dui_main"};
```

Comment:
We still use the speaking variable here. So we still need this.

Reply:
```sqf
_unit getVariable [QEGVAR(radar,isSpeaking), 0]
```
soft dependency.

Reply:
Yeah, I would keep it as long as we need it. Or should we move that speaking thing to the main?

Reply:
maybe for the next release. Not feeling moving that out atm.

Reply:
then i say keep the dependency ^^

Reply:
bleh